win/WIN_ColourValueTextbox.h

```diff
@@ -19,12 +19,12 @@ namespace win
 		void redirectLinkedVariable(uint8_t* newVariable);
```

Comment:
What does this do?

Reply:
intended to allow change of the linked variable, but realised there should be no cases where this happens for ColourValueTextbox, so removing